metadeploy/api/views.py

```diff
@@ -207,6 +208,20 @@ def preflight(self, request, pk=None):
         if request.method == "POST":
             return self.preflight_post(request)
 
+    @action(detail=True, methods=["post"])
```

Comment:
@wlonk This endpoint seems to require authentication, but I don't think it should.

`401: {"detail":"Authentication credentials were not provided."}`